cpp/include/cudf/search.hpp

```diff
@@ -99,5 +99,23 @@ gdf_column upper_bound(table const& t,
                        std::vector<bool> const& desc_flags,
                        bool nulls_as_largest = true);
 
+/**---------------------------------------------------------------------------*
+ * @brief Find if the `value` is present in the `column`
+ *
```

Comment:
Need to document the requirement that `value` and `column` have the same `dtype`.

Reply:
@jrhemstad Should I also throw an exception for this case in function body?

Reply:
> this supports if the column is a int and value is float and vice versa.

I'm not following. The function as it is implemented now can only support `column.dtype == scalar.dtype`.

Reply:
Ok, I will take care of it.